### PR TITLE
Internal hostname gateway follow up

### PR DESCRIPTION
**Pull request by Jooho**
## Description

The original PR was accidentally merged so I create another pr to follow up the ticket.(https://github.com/opendatahub-io/odh-model-controller/pull/221)

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Review by israel-hdez — COMMENTED** [review]
Just one small fix, and then I think this is ready to go!

File: go.sum [hunk 1]
```diff
@@ -6,86 +6,268 @@ cloud.google.com/go v0.44.2/go.mod h1:60680Gw3Yr4ikxnPRS/oxxkBccT6SA1yMk63TGekxK
 cloud.google.com/go v0.44.3/go.mod h1:60680Gw3Yr4ikxnPRS/oxxkBccT6SA1yMk63TGekxKY=
 cloud.google.com/go v0.110.10 h1:LXy9GEO+timppncPIAZoOj3l58LIU9k+kn48AN7IO3Y=
 cloud.google.com/go v0.110.10/go.mod h1:v1OoFqYxiBkUrruItNM3eT4lLByNjxmJSV/xDKJNnic=
+cloud.google.com/go/accessapproval v1.7.4/go.mod h1:/aTEh45LzplQgFYdQdwPMR9YdX0UlhBmvB84uAmQKUc=
```
[the remark this answers is not in the document]
Comment (Jooho) [reply go.sum]: interesting, why keep changing this ? I think we should add git pre commit or Make record to do `go mod tidy` by default. anyway I updated for now

File: controllers/reconcilers/kserve_isvc_gateway_reconciler.go [hunk 2]
```diff
@@ -237,7 +238,10 @@ func (r *KserveGatewayReconciler) copyServingCertSecretFromIsvcNamespace(ctx con
 			Name:      fmt.Sprintf("%s-%s", sourceSecret.Name, sourceSecret.Namespace),
 			Namespace: meshNamespace,
 			Labels: map[string]string{
-				"opendatahub.io/managed": "true",
```
[the remark this answers is not in the document]
Comment (Jooho) [reply controllers/reconcilers/kserve_isvc_gateway_reconciler.go:240]: Right, it needs the label. Added

**Review by israel-hdez — APPROVED** [review]
/lgtm

**Pull request merged**